api/query.py

```diff
@@ -47,6 +47,20 @@ class Location(NamedTuple):
     stop: Position
 
 
+class Type(NamedTuple):
+    location: Dict[str, Position]
+    annotation: str
+
+    def extract_function_model(self) -> str:
+        results = re.findall("\(.*?\)", self.annotation)
```

Comment:
The annotation will look something like this:
```
"typing.Callable(views.operate_on_twos)[[Named(request, django.http.request.HttpRequest)], django.http.response.HttpResponse]"
```

It's probably a good idea to be a bit more strict on the validation (maybe require matching the preceding `typing.Callable`?)

Reply:
Also note that right now you're just returning `views.operate_on_twos` from that example, but you really need to be returning `def views.operate_on_twos(request): ...`